src/utils/client.js

```diff
@@ -36,10 +36,14 @@ function createCtpClient({
     contactEmail: packageJson.author.email,
   })
 
+  const arrayFrom500To509 = Array.from({ length: 100 }, (ignore, i) => i + 500)
```

Comment:
I could, but I think it doesn't harm anyway :) And we specify it in our docs that on 5xx we retry, so I would stick to it since this doc will be in the hand of customers.

Reply:
But you are running >100 test to just to be safe, and commercetools is not returning those status codes 🤔 (https://docs.commercetools.com/api/errors)

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/10/commits/880ea9e4ca2a13fbfc44b107077925940d714c3d

Reply:
Please update the array in unit test too 🙏🏽

Reply:
You mean this test, right? https://github.com/commercetools/commercetools-subscriptions/pull/10/files#diff-a482c8b979b90b008276c19a5f7bf9420873fd78a088519137b6b4a0f1d50ae7R179

This test is a bit different. It tests the returned codes from external URL and not from CTP. And in our concept, we defined following:
```
200: successful processing
5xx, 401, 403: retry
Network issues will lead the cron job to retry on the subsequent runs
All other HTTP status code will set the template-order state to Error which then needs to be handled (i.e. reactivated after manual fix).
```